src/NuGet.Licenses/Services/LicenseFileService.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Web;
+namespace NuGet.Licenses
+{
+    public class LicenseFileService : ILicenseFileService
+    {
+        public string GetLicenseFilePath(string licenseIdentifier)
+        {
+            return Path.GetFullPath(Path.Combine(HttpContext.Current.Server.MapPath("~\\App_Data\\Licenses\\"), String.Concat(licenseIdentifier, ".txt")));
```

Comment:
We'll have to test this code and in its current form it is quite untestable. I suggest wrapping all file I/O calls into their own method, so we'd have something like:
```C#
interface IFileService
{
    string GetFullPath(string relativePath);
    bool FileExists(string path);
    string ReadAllText(string path);
}
```
And separately, move `HttpContext` operations into it's own object, so for tests we wouldn't have to construct HttpContext, but mock that object instead.

Reply:
Create a new service to get folder path.

Reply:
Need file operations abstractions as well.